interfaces/cython/cantera/composite.py

```diff
@@ -1306,6 +1327,9 @@ def _make_functions():
     def empty_scalar(self):
         return np.empty(self._shape)
 
+    def empty_strings(self):
+        return np.empty(self._shape, dtype='<U14')
```

Comment:
With this value hardcoded to 14 characters, anything additional is truncated off the value. I'm not sure if there's a way to work around that, but this would be an extremely tricky bug to track down if we ever have strings longer than 14 characters for phases of matter. If there's not an easy way to resolve it, then maybe just setting it to a much larger number would be sufficient (say 50?).

I also wonder how this would affect `extra` columns that are longer than 14 characters?

Reply:
Was wondering about this myself (but settled on the longest string currently returned by phase-of-matter). I don’t think that this line affects `extra`, but it would affect potential future string attributes. I don’t have a problem making it longer ...

Reply:
I increased the length to 50; there is a work-around suggested [here](https://stackoverflow.com/questions/9108837/numpy-recarray-strings-of-variable-length) - i.e. use an array of `object` type, rendering string length flexible - but that will open a can of worms elsewhere when it comes to saving, restoring, etc..